apps/antalmanac/src/stores/ThemeStore.ts

```diff
@@ -0,0 +1,43 @@
+import { create } from 'zustand';
```

Comment:
I'm of the opinion that there should be a `settingsStore.ts`, where `themeStore` and whatever the time preference store is contained. Splitting it like this makes it a little fractured, but I don't feel so strongly about it.

Reply:
I agree; implemented